### PR TITLE
TW-1048 Super Admin role assigment changes

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 🌟

**Comment by johnoffenhartzPM** [comment]
Looks great!

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 1]
```diff
@@ -138,6 +139,18 @@ Teams can have two support accounts at no extra cost. Support accounts are team
 
 For full details about the Postman team roles that you can assign as a Team Admin, see [Team roles](/docs/collaborating-in-postman/roles-and-permissions/#team-roles). Some roles are only available for teams with [Postman Professional or Enterprise](https://www.postman.com/pricing) plans.
 
+#### Managing super admins
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:142]:
```suggestion
#### Managing Super Admins
```

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ With the [Admin role](/docs/collaborating-in-postman/roles-and-permissions/#team
...
 
-* **Super Admin** ([Enterprise plans only](https://www.postman.com/pricing)) - Manages everything within a team, including team settings, members, roles, and resources in [public, team, or private workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility). Team members with this role can perform all actions that Admin, Billing, Community Manager, API Network Manager, Developer, and Partner Manager roles can perform.
+* **Super Admin** ([Enterprise plans only](https://www.postman.com/pricing)) - Manages everything within a team, including team settings, members, roles, and resources in [public, team, or private workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility). Team members with this role can perform all actions that Admin, Billing, Community Manager, API Network Manager, Developer, and Partner Manager roles can perform. For information on assigning this role, see [Managing super admins](/docs/administration/managing-your-team/managing-your-team/#managing-super-admins).
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:57]:
```suggestion
* **Super Admin** ([Enterprise plans only](https://www.postman.com/pricing)) - Manages everything within a team, including team settings, members, roles, and resources in [public, team, or private workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility). Team members with this role can perform all actions that Admin, Billing, Community Manager, API Network Manager, Developer, and Partner Manager roles can perform. For information on assigning this role, see [Managing Super Admins](/docs/administration/managing-your-team/managing-your-team/#managing-super-admins).
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**